backend/rest/authRoutes.ts

```diff
@@ -167,4 +168,23 @@ authRouter.post(
   },
 );
 
+/* Reset password through a "Forgot Password" option */
+authRouter.post(
+  "/forgotPassword",
+  forgotPasswordRequestValidator,
+  async (req, res) => {
+    try {
+      try {
+        await userService.getUserByEmail(req.body.email);
+        await authService.resetPassword(req.body.email);
+        res.status(204).send();
+      } catch (error) {
+        res.status(401).json({ error: "EMAIL_NOT_FOUND" });
+      }
+    } catch (error: unknown) {
```

Comment:
Is this outer try/catch block actually being used? With the current setup, it seems like it would never be triggered because all the code is wrapped in the inner try block. As a result, any error—whether it's a 401 or 500—would be caught by the inner catch block. Since the inner catch block handles the exception, it effectively marks it as resolved. There would be no unhandled exceptions left for the outer try/catch block to catch, rendering it redundant / unused.